qml/MainMenu.qml

```diff
@@ -166,20 +166,64 @@ Item {
                     enabled: !autoGainChB.checked
                     visible: SettingsManager.enableSensorAdvancedConfiguration
 
+                    property var maxLengthMm: SettingsManager.debugMode ? 1e6 : 5e4
+                    property var minLengthMm: 1e3
+
+                    function setDepth(startMm, lengthMm) {
+                        // Sensor max range is limited to maxLengthMm, this value is also the max window depth
+                        // The minimum range between start and length is minLengthMm
+
+                        // Start
+                        if(ping.start_mm !== startMm) {
+                            // Check if length + new start is bigger than our limit (maxLengthMm)
+                            if(startMm >= maxLengthMm - minLengthMm) {
```

Comment:
Should you be using `maxDepthMm` where you are using `maxLengthMm`?

Reply:
maxDepthMm is the correct variable, something was lost while squashing